### PR TITLE
feat:Real-time data access

### LINKED ISSUE #512: data processing task : Real-time data access
# What
   Crawl URL content(text、image、table ... eg) manually or periodically
   
# How
   Web crawling using Playwright. Scheduling was using Kubernetes CronJob   
   
# Goal
   After craw the url content, you can add the content to the knowledge base.

### PR DESCRIPTION
**Pull request by GHOSTHSP**
#### What this PR does / why we need it
Real-time data access

#### Which issue(s) this PR fixes

Fixes #512

### CODE REVIEW

File: deploy/charts/arcadia/templates/pg-init-data-configmap.yaml [hunk 1]
```diff
@@ -326,7 +326,50 @@ data:
         COMMENT ON COLUMN public.data_process_task_stage_log.update_datetime IS '更新时间';
         COMMENT ON COLUMN public.data_process_task_stage_log.update_user IS '更新人';
         COMMENT ON COLUMN public.data_process_task_stage_log.update_program IS '更新程序';
+        
+        create table if not exists data_process_task_document_web_url
```
Comment (bjwswang) [inline deploy/charts/arcadia/templates/pg-init-data-configmap.yaml:330]: should use uppercase here.

**Review by wangxinbiao — APPROVED** [review]

File: deploy/charts/arcadia/templates/pg-init-data-configmap.yaml [hunk 2]
```diff
@@ -326,7 +326,50 @@ data:
         COMMENT ON COLUMN public.data_process_task_stage_log.update_datetime IS '更新时间';
         COMMENT ON COLUMN public.data_process_task_stage_log.update_user IS '更新人';
         COMMENT ON COLUMN public.data_process_task_stage_log.update_program IS '更新程序';
+        
+        CREATE TABLE IF NOT EXISTS data_process_task_document_web_url
```
Comment (bjwswang) [inline deploy/charts/arcadia/templates/pg-init-data-configmap.yaml:330]: @GHOSTHSP @wangxinbiao  Please note that we should alway upgrade the chart version once you update it

File: pypi/data-processing/requirements.txt [hunk 3]
```diff
@@ -22,3 +22,7 @@ opencc-python-reimplemented==0.1.7
 selectolax==0.3.17
 openai==1.3.7
 python-docx==1.1.0
+
+bs4==0.0.1
+playwright=1.40.0
```
Comment (bjwswang) [inline pypi/data-processing/requirements.txt:27]: @GHOSTHSP  You have introduced new dep `playwright `. What is this for?

Comment (GHOSTHSP) [reply pypi/data-processing/requirements.txt:27]: To handle websites built with React 、Vue or ... , simulate a browser to scrape web page data

File: deploy/charts/arcadia/templates/pg-init-data-configmap.yaml (hunk 2 above)
Comment (wangxinbiao) [reply deploy/charts/arcadia/templates/pg-init-data-configmap.yaml:330]: @GHOSTHSP  upgrade the chart version. deploy/charts/arcadia/Chart.yaml

Comment (GHOSTHSP) [reply deploy/charts/arcadia/templates/pg-init-data-configmap.yaml:330]: The version has been updated to 0.1.29

File: pypi/data-processing/src/database_operate/dp_document_image_db_operate.py [hunk 4]
```diff
@@ -0,0 +1,236 @@
+# Copyright 2023 KubeAGI.
```
Comment (bjwswang) [inline pypi/data-processing/src/database_operate/dp_document_image_db_operate.py:1]: It's `2024` now  😆

File: pypi/data-processing/src/utils/web_url_utils.py [hunk 5]
```diff
@@ -0,0 +1,620 @@
...
+from bs4 import BeautifulSoup
+from PIL import Image
+from playwright.sync_api import sync_playwright
+
+from common import log_tag_const
+from utils import date_time_utils
+
+logger = logging.getLogger(__name__)
+
+def handle_website(
```
Comment (bjwswang) [inline pypi/data-processing/src/utils/web_url_utils.py:33]: @GHOSTHSP @wangxinbiao Can we implement a  new `PlaywrightURLLoader` which extends the langchain PlaywrightURLLoader?

Comment (bjwswang) [reply pypi/data-processing/src/utils/web_url_utils.py:33]: 目前这种实现方式太松散，需要整合出一个完整的Web加载器功能

**Review by bjwswang — CHANGES_REQUESTED** [review]
See my comments

**Review by bjwswang — APPROVED** [review]

**Pull request merged**